packages/manager/cypress/e2e/core/placementGroups/update-placement-group-label.spec.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * @file Integration tests for Placement Group update label flows.
+ */
+
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import { randomLabel, randomNumber } from 'support/util/random';
+import {
+  mockGetPlacementGroups,
+  mockUpdatePlacementGroupLabel,
+  mockUpdatePlacementGroupLabelError,
+} from 'support/intercepts/placement-groups';
+import { accountFactory, placementGroupFactory } from 'src/factories';
+import { mockGetAccount } from 'support/intercepts/account';
+import { authenticate } from 'support/api/authentication';
+import type { Flags } from 'src/featureFlags';
+import { chooseRegion } from 'support/util/regions';
+import { ui } from 'support/ui';
+import type { UpdatePlacementGroupPayload } from '@linode/api-v4';
+
+const mockAccount = accountFactory.build();
+authenticate();
+
+describe('Placement Group update label flow', () => {
+  // Mock the VM Placement Groups feature flag to be enabled for each test in this block.
+  beforeEach(() => {
+    mockAppendFeatureFlags({
+      placementGroups: makeFeatureFlagData<Flags['placementGroups']>({
+        beta: true,
+        enabled: true,
+      }),
+    });
+    mockGetFeatureFlagClientstream();
+    mockGetAccount(mockAccount).as('getAccount');
+  });
+
+  /**
+   * - Confirms that a Placement Group's label can be updated from the landing page.
+   * - Confirms that clicking "Edit" opens PG edit drawer.
+   * - Only the label field is shown in the edit drawer.
+   * - New label can be entered into the label field.
+   * - Confirms that Placement Groups landing page updates to reflect successful label update.
+   * - Confirms a toast notification is shown upon successful label update.
+   */
+  it("update to a Placement Group's label is successful", () => {
+    const mockPlacementGroupCompliantRegion = chooseRegion();
+
+    const mockPlacementGroup = placementGroupFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+      region: mockPlacementGroupCompliantRegion.id,
+      affinity_type: 'anti_affinity:local',
+      is_compliant: true,
+      is_strict: false,
+      members: [],
+    });
+
+    const newLabel: UpdatePlacementGroupPayload = {
+      label: randomLabel(),
+    };
```

Comment:
```suggestion
    const mockPlacementGroupUpdated = {
      ...mockPlacementGroup,
      label: randomLabel(),
    };
```
This is going to be the mock data that the API responds with upon updating the label, so we'll want it to include the full PG object!

(There are a few instances where `newLabel.label` is used later in the test that would need to be updated too to reflect the const name change)

Reply:
Perfect, this makes total sense to me now. Thanks @jdamore-linode